packages/expo-video/android/src/main/java/expo/modules/video/VideoView.kt

```diff
@@ -249,6 +252,7 @@ class VideoView(context: Context, appContext: AppContext) : ExpoView(context, ap
         .commitAllowingStateLoss()
     }
     applyAutoEnterPiP(currentActivity, autoEnterPiP)
+    this.playerView.useController = useNativeControls
```

Comment:
Can we move this to `OnViewDidUpdateProps`? It might be too early, but I would like to give it a shot. Applying it here seems odd.

Reply:
I'll try it out